### ISSUE TITLE
audit(agn): the plbendwidth prior and GRAHSP's stated range for the same dimensionless Lambda intersect at one endpoint

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#2159]
## Summary

tengri's prior for the smooth-bending-power-law bend width `Λ` and GRAHSP's stated range for the same dimensionless quantity are very nearly disjoint. The formula is identical on both sides; only the prior differs, and the divergence is not recorded anywhere.

This may well be the right call — upstream's guidance for this parameter is demonstrably confused — but right now it reads as an accident rather than a decision.

## The numbers

| | declared | stated usable range |
|---|---|---|
| GRAHSP `plbendwidth` (`activatepl.py:90-95`) | default `10.0` | "10 to 1000 nm are reasonable" |
| tengri `agn_grahsp_plbendwidth` (`_params.py:624-629`) | `Uniform(0.1, 10.0, default=1.0)` | "Typical 0.1-10" [dex] |

The defaults differ by 10x, and the two ranges intersect only at the single endpoint `10.0`. A tengri fit explores bend widths upstream considers below its usable range, and cannot reach almost any of the range upstream recommends.

## The formula is the same, so this is purely a prior question

Both sides implement the Ryde parameterization with the same natural-log convention:

```python
# GRAHSP  activatepl.py:49-68
q = log(x/xbrk) / Lambda
... ** ((lam2 - lam1)/2. * Lambda)

# tengri  bbb.py:142-145
q = jnp.log(wave / plbendloc_nm) / plbendwidth
sub_expo = (plslope - uvslope) / 2.0 * plbendwidth
```

`log(x/xbrk)` is a ratio, so `Λ` must be dimensionless for `q` to mean anything, and it appears again multiplying a dimensionless exponent. tengri documents this correctly (`bbb.py:114`: "Bend width :math:`\Lambda` (dimensionless)").

**Upstream's docstring is wrong about the units**: it says "Width of the powerlaw bend, in nm / 10 to 1000 nm are reasonable". A quantity that divides a log ratio has no nm. So upstream's stated range is attached to a unit the parameter does not have, which is good reason to distrust the range itself — and plausibly the reason our prior was chosen differently.

## Why it still needs resolving

`Λ` is not a nuisance knob. It sets how sharply the disc continuum bends between the UV and optical slopes, so the prior determines which continuum shapes are reachable at all. Two possibilities:

1. The prior was deliberately recalibrated because upstream's guidance is unusable. Then the reasoning belongs in the declaration or a note, so the next person comparing against GRAHSP does not read it as a parity failure.
2. The prior was set independently and the divergence was never noticed. Then it is worth asking whether `[0.1, 10]` covers the shapes we need, given upstream places its default at the top edge of our range.

Either way the current state is a silent divergence from the reference implementation on a parameter where we otherwise match it exactly.

This is the shape of #1586 — a declared prior that does not line up with the region the model was built for produces a well-behaved fit over the wrong space, with nothing to indicate it.

## Context

#333 introduced this model and describes the parameter as "bend steepness" / "smoothness of the break (positive)" — correctly claiming no units, but without recording a range decision. The upstream default does not appear to have been considered at that point.

## Scope

- [ ] Decide whether `Uniform(0.1, 10.0, default=1.0)` is intended, and record why if so
- [ ] If it is a recalibration, state in the declaration that it deliberately departs from upstream's stated range and why upstream's is unreliable
- [ ] Sanity-check that the reachable continuum shapes over `[0.1, 10]` cover the SDSS quasar diversity #333 cited as the motivation
- [ ] Confirm nothing in `tests/components/agn/grahsp/` pins parity at a `Λ` outside our own prior (the parity fixture reads `plbendwidth` from upstream values; other tests use `1.0`)

## Note

Checked at GRAHSP `45054dd`. I have not verified whether upstream has changed this parameter since our implementation was written; that is worth a look as part of resolving it.

## Related

- #2158 — the `agn_grahsp_a_bc` reference-wavelength defect found in the same parity pass.

**Comment by suchethac** [comment]
## Resolved — bound widened to 100, default kept at 1.0

Measured rather than argued. Verdict rule fixed before running: sensitivity of the amplitude-normalized SBBB shape to `Lambda` over 50-1000 nm rest frame; if the shape moves <1% across a decade of `Lambda` that decade is inert and our bound stands, if >10% the bound truncates reachable models.

Probe: `sbpl_bbb(uvslope=0.0, plslope=-1.7, plbendloc_nm=100.0)`, shape normalized at 510 nm, compared against the `Lambda -> inf` limit (a pure power law).

| Lambda | max abs deviation vs `Lambda -> inf` | vs 10x larger |
|---:|---:|---:|
| 0.1 | 276.6% | 64.6% |
| 1.0 | 128.8% | 104.5% |
| 10.0 | **11.9%** | **10.6%** |
| 100.0 | 1.13% | 1.02% |
| 1000.0 | 0.113% | 0.102% |

The deviation falls as `~1/Lambda`, which is what the algebra gives: for large `Lambda` the correction exponent goes as `delta_alpha * (ln^2(lam/lam_brk) - ln^2(lam_0/lam_brk)) / (4 Lambda)`.

**Verdict.** At `Lambda = 10` the shape still moves 10.6% across the next decade — above the 10% line, so the old upper bound truncated reachable model space. Sensitivity drops below 1% only near `Lambda = 100`; by 1000 it is 0.11%, undetectable at any realistic photometric S/N.

So both readings in the issue were partly right. Upstream's "10 to 1000" *is* mostly inert above ~100, which is why our range was skeptical of it — but our bound was in the wrong place, and the sharper problem stands: **upstream's default of 10.0 sat exactly on our prior edge**, where a sampler has a one-sided gradient and cannot explore around it.

## Change

`components/agn/_params.py` — `Uniform(0.1, 10.0, default=1.0)` -> `Uniform(0.1, 100.0, default=1.0)`.

Widening only. The default stays 1.0, so no existing fit changes result — unless it was railing at the old bound, in which case it was already wrong. Moving the default to upstream's 10.0 *would* change results and is a separate decision, not made here.

The declaration now records the measured numbers, the inertness threshold, and the deliberate divergence from upstream's default. It also corrects the units label from `[dex]` to dimensionless, matching `bbb.py:114` — `Lambda` divides a natural log, so it is not in dex either. That second slip is the same class as #2158.

## Verification

`tests/components/agn/grahsp/` 75 passed, 3 skipped. Full `tests/components/agn` 631 passed, 57 skipped, 6 xfailed. `ruff check` and `ruff format --check` clean. `check_british_spelling`, `check_doc_examples`, `check_param_prefixes`, `check_reimplementation_language` all pass.

Still open from the issue: whether upstream has changed this parameter since our implementation was written. Measured against GRAHSP `45054dd`.
